public/index.html

```diff
@@ -4,17 +4,46 @@
-        <input type="button" value="Bin to Dec" onclick="bin2Dec(document.getElementById('num').value)"/>
-        <input type="button" value="Dec to Bin" onclick="dec2Bin(document.getElementById('num').value)"/>
-        <input type="button" value="Bin to Hex" onclick="bin2Hex(document.getElementById('num').value)"/>
-        <input type="button" value="Hex to Bin" onclick="hex2Bin(document.getElementById('num').value)"/>
-        <input type="button" value="Bin to Oct" onclick="bin2Oct(document.getElementById('num').value)"/>
-        <input type="button" value="Oct to Bin" onclick="oct2Bin(document.getElementById('num').value)"/>
-        <p id="conver">Resultado</p>
+        <div id="container">
+            <label for="num">Enter the value to convert:</label>
+            <input type="text" id="num" name="name" required minlength="1" maxlength="16" size="20" />
+            <div class="button-group">
+                    <p>From binary to:</p>
+                <div id="bin">
+                    <input type="button" value="Decimal" onclick="bin2Dec(document.getElementById('num').value)"/>
+                    <input type="button" value="Hexadecimal" onclick="bin2Hex(document.getElementById('num').value)"/>
+                    <input type="button" value="Octal" onclick="bin2Oct(document.getElementById('num').value)"/>
+                </div>
+            </div>
+            <div class="button-group">
+                <p>From decimal to:</p>
+                <div id="dec">
+                    <input type="button" value="Binary" onclick="dec2Bin(document.getElementById('num').value)"/>
+                    <input type="button" value="Hexadecimal" onclick="dec2Hex(document.getElementById('num').value)"/>
+                    <input type="button" value="Octal" onclick="dec2Oct(document.getElementById('num').value)"/>
+                </div>
+            </div>
+            <div class="button-group">
+                <p>From hexadecimal to:</p>
+                <div id="hex">
+                    <input type="button" value="Binary" onclick="hex2Bin(document.getElementById('num').value)"/>
+                    <input type="button" value="Decimal" onclick="hex2Dec(document.getElementById('num').value)"/>
+                    <input type="button" value="Octal" onclick="hex2Oct(document.getElementById('num').value)"/>
+                </div>
+            </div>
+            <div class="button-group">
+                <p>From Octal to:</p>
+                <div id="oct">
+                    <input type="button" value="Binary" onclick="oct2Bin(document.getElementById('num').value)"/>
+                    <input type="button" value="Decimal" onclick="oct2Dec(document.getElementById('num').value)"/>
+                    <input type="button" value="Hexadecimal" onclick="oct2Hex(document.getElementById('num').value)"/>
+                </div>
+            </div>
+            <p id="conver">Resultado</p>
```

Comment:
The use of a generic id like "conver" for the output might be confusing. Consider a more descriptive name like "conversionOutput".

Reply:
Yep, same as the calc-opt or except-handl issue, just using short names, but I edited as suggested cause I think is clearer and a better way to understand it.

thanks for your comment
https://github.com/GonzoVM/bin2dec/commit/de058f42ab51c954cf95330ddd3cf7e615da85a7